src/controllers/image_controller.py

```diff
@@ -0,0 +1,31 @@
+import ttkbootstrap as tb
+from src.widgets import image_widget as iw
+
+
+# Create Image Controller Frame
+class ImageController(tb.Frame):
+    gridX = 0
+    gridY = 0
+
+    open_images = []
+
+    def __init__(self, parent, root):
```

Comment:
What's the difference between parent and root in this context?

Reply:
The root refers to the main window of the app thats defined in the MainWindow whereas as parent refers to the direct parent of the current widget. Using the image controller as an example it is a child of the horizontal panned window, so in this case parent would refer to the horizontal panned window while root is still a reference to the MainWindow

Reply:
I see - is that meant to be passed in here: https://github.com/mitchell-merry/emu-viewer/pull/12/files#diff-2e5ad92c43aa96cc3a9cef6c6aec998b216f1379c43b1f651013d25e55989312R52

Reply:
Yeah its passed in there and because the ContentWindow's parent is the main window and then that is passed through to the image controller as the root window, I probably need to rename it to make it a bit clearer, looking at it now, any reference to the main window should probably just called root